projects/arlas-toolkit/assets/i18n/en.json

```diff
@@ -1,300 +1,303 @@
+	"Registration validated, you will receive an email to validate your account": "Registration validated, you will receive an email to validate your account",
+	"Registration is not enabled": "Registration is not enabled.",
+	"No search": "Two characters or more are needed.",
+	"Searching": "Searching...",
+	"No results": "No results matching the search.",
+	"Remove search text": "Remove search text",
+	"Mode": "Mode",
+	"Desired precision": "Desired precision",
+	"None": "None",
+	"Order by": "Order by",
+	"Types": "Types",
+	"are excluded": "are excluded",
+	"Downloading Geojson file": "Downloading Geojson file",
+	"Downloading Shapefile": "Downloading Shapefile",
+	"Download Geojson": "Download Geojson",
+	"Click to download the chosen layer as a Geojson file": "Click to download the chosen layer as a Geojson file",
+	"Download Shapefile": "Download Shapefile",
+	"Click to download the chosen layer as a zipped Shapefile": "Click to download the chosen layer as a zipped Shapefile",
+	"Operation name": "Operation name",
+	"Field to tag": "Field to tag",
+	"Value of tag": "Value of tag",
+	"With propagation": "With propagation",
+	"On field": "On field",
+	"Link to": "Link to",
+	"Add Tag": "Add Tag",
+	"Remove Tag": "Remove Tag",
+	"Operation Name": "Operation Name",
+	"Field tagged": "Field tagged",
+	"Propagation field": "Propagation field",
+	"Action": "Action",
+	"Replay": "Replay",
+	"Datepicker.rangeFromLabel": "From",
+	"Datepicker.rangeToLabel": "To",
+	"Time zone": "Time zone:",
+	"Remove time filter": "Remove time filter",
+	"No data available.": "No data available.",
+	"no access to apps": "You don't have access to other apps.",
+	"About": "About",
+	"Go to": "Go to",
+	"My account": "My account",
+	"Your profile": "My profile",
+	"Change my password": "Change my password",
+	"Logout": "Logout",
+	"User informations": "User informations",
+	"Organisations": "Organisations",
+	"Verify your email": "Verify your email",
+	"To validate your email, please set your password:": "To validate your email, please set your password:",
+	"Your account is now active.": "Your account is now active.",
+	"Unknown user": "Unknown user",
+	"User already verified": "User already verified",
+	"An error occured": "An error occured",
+	"Validate": "Validate",
+	"native": "Native",
+	"Bookmark already exists": "A bookmark already exists with this name.",
+	"Bookmark creation error": "Error during bookmark creation.",
+	"Bookmark succefully created.": "Bookmark succefully created.",
+	"Unauthorized to create a dashboard, you need to log in": "Unauthorized to create a dashboard, you need to log in",
+	"Missing permissions to create a dashboard": "Missing permissions to create a dashboard",
+	"A configuration with this name exists already, please choose another name": "A dashboard with this name exists already, please choose another name",
+	"An error occurred, please try later": "An error occurred, please try later",
+	"Loading data of": "Loading data of",
```

Comment:
Shouldn't there be a variable for this string ?

Reply:
I agree but it is not how it is implemented , so i prefer to treat it in another PR